app/utils/fileDownload/index.ts

```diff
@@ -0,0 +1,59 @@
+import RNFetchBlob, { FetchBlobResponse } from 'rn-fetch-blob';
+import FileViewer from 'react-native-file-viewer';
+
+import EventEmitter from '../events';
+import { LISTENER } from '../../containers/Toast';
+import I18n from '../../i18n';
+import { DOCUMENTS_PATH, DOWNLOAD_PATH } from '../../constants/localPath';
+
+interface IAttachment {
+	title: string;
+	title_link: string;
+	type: string;
+	description: string;
+}
+
+export const getExtensionType = (text: string): string | undefined => text.split('.').pop();
+
+export const getLocalFilePathFromFile = (localPath: string, attachment: IAttachment): string => {
+	const fileName = attachment.title.split('.')[0];
+	return `${localPath}${fileName}.${getExtensionType(attachment.title_link)}`;
```

Comment:
I didn't get why we need to do this.
Wouldn't this work `${localPath}${attachment.title}` without getting the extension type?

Reply:
Great question! 
I created this function, because I preferred get the extension from the `attachment.title_link`, because  in my thought Is not very safe get this information from the name of the file. The best idea shoud be get a new field `extensions` from the API. 

But, after your comment I stopped to think about, and this function:

```const fileName = attachment.title.split('.')[0];```

was not safe either, because if the file has a `.` in the file name, will crash our app. 

So I just changed the code as you suggested, thanks